<launch>
    <arg name="control_interface" default="twist"/>
    <arg name="example"           default="false"/>
    <arg name="params_yaml_file"  default="$(find iiwa_interactive_controller)/config/passiveDS_impedance_params.yaml" />

    <include file="$(find iiwa_gazebo)/launch/iiwa_gazebo.launch">
      <arg name="gui" value="true"/>
      <arg name="model" value="14" />
    </include>

    <node name="iiwa_interactive_bringup" pkg="iiwa_interactive_controller" type="iiwa_interactive_controller_bringup" output="screen" cwd="node" />

    <group if="$(arg example)">
        <node name="iiwa_interactive_position_interface" pkg="iiwa_interactive_controller" type="iiwa_interactive_controller_position_interface" output="screen" cwd="node" />
    </group>

    <param name="control_interface"   value="$(arg control_interface)"/>
    <rosparam file="$(arg params_yaml_file)" command="load"/>
</launch>